app/_includes/md/mesh/install-universal-run.md

```diff
@@ -1,18 +1,18 @@
 <!-- Shared between Mesh installation topics: Ubuntu, Amazon Linux, RedHat, Debian, MacOS, CentOS -->
 ## 2. Run {{site.mesh_product_name}}
 
-Once downloaded, you will find the contents of {{site.mesh_product_name}} in the `kong-mesh-{{include.kong_latest.version}}` folder. In this folder, you will find &mdash; among other files &mdash; the bin directory that stores all the executables for {{site.mesh_product_name}}.
+Once downloaded, you will find the contents of {{site.mesh_product_name}} in the `kong-mesh-{{page.kong_latest.version}}` folder. In this folder, you will find &mdash; among other files &mdash; the bin directory that stores all the executables for {{site.mesh_product_name}}.
```

Comment:
You shouldn't use `page` inside an include. To make it work correctly, the include statements in the installation topics should look like this:

```
{% include_cached /md/mesh/install-universal-run.md kong_version=page.kong_version %}

{% include_cached /md/mesh/install-universal-verify.md kong_version=page.kong_version  %}

{% include_cached /md/mesh/install-universal-quickstart.md kong_version=page.kong_version %}
```